src/lib.rs

```diff
@@ -47,6 +49,68 @@ impl SourceMapGenConfig for SourceMapConfig {
     }
 }
 
+#[derive(Default, Debug)]
+struct ContentTagVisitor {
+    occurrences: Vec<Occurrence>,
+}
+
+impl ContentTagVisitor {
+    fn add_occurrence(
+        &mut self,
+        span: &Span,
+        opening: &ContentTagStart,
+        contents: &ContentTagContent,
+        closing: &ContentTagEnd,
+    ) {
+        let occurrence = Occurrence {
+            range: span.into(),
+            content_range: contents.span.into(),
+            end_range: closing.span.into(),
+            start_range: opening.span.into(),
+            tag_name: "template".to_owned(),
+            kind: "template-tag".to_owned(),
+            contents: contents.value.to_string(),
+        };
+
+        self.occurrences.push(occurrence);
+    }
+}
+
+impl Visit for ContentTagVisitor {
+    fn visit_expr(&mut self, n: &swc_ecma_ast::Expr) {
+        match n {
+            swc_ecma_ast::Expr::ContentTagExpression(ContentTagExpression {
+                span,
+                opening,
+                contents,
+                closing,
+            }) => {
+                self.add_occurrence(span, opening, contents, closing);
+            }
+            _ => {}
+        }
+    }
+
+    fn visit_class_member(&mut self, n: &ClassMember) {
+        match n {
+            ClassMember::ContentTagMember(ContentTagMember {
+                span,
+                opening,
+                contents,
+                closing,
+            }) => {
+                self.add_occurrence(span, opening, contents, closing);
+            }
+            _ => {}
+        }
+    }
+
+    // FIXME: is this already covered by visit_class_member?
```

Comment:
I think the issue is we need to update something on our swc fork for the visitor to cover the content tag member, it's on my TODO, but this is fine for now

Reply:
`fn visit_class_members` here is superfluous. 

But I think perhaps you are missing the necessary recursive visits in both visit hooks. A test case that would reveal this is a content tag expression embedded within another expression, as in: https://github.com/embroider-build/content-tag/blob/227d58df47a27b916a9f84dfc54c40082558b8eb/src/transform.rs#L212-L221

Similarly a content tag expression inside the body of a class method would reveal whether the class member visitor is recursing properly.

In our existing visitor the recursion happens here:

https://github.com/embroider-build/content-tag/blob/227d58df47a27b916a9f84dfc54c40082558b8eb/src/transform.rs#L69

and here:

https://github.com/embroider-build/content-tag/blob/227d58df47a27b916a9f84dfc54c40082558b8eb/src/transform.rs#L77

Reply:
Recursive visitor implemented in  f9ae9c2d706309756bef94e98d92dbad76fa0ffe